src/cnxml/serialize.js

```diff
@@ -75,6 +75,14 @@ const BLOCK_TAGS = {
 }
 
 
+/**
+ * List of valid declensions for xrefs.
+ * 
+ * @see xref
+ */
+const DECLENSIONS = ['genitive', 'dative', 'accusative', 'instrumental', 'locative', 'vocative']
```

Comment:
You shouldn't limit it to just cases used in Polish. If you want a list of all existing grammatical cases (at least according to Wikipedia) I should still have it somewhere.

Reply:
Sure, please send me if you find it or feel free to add it

Reply:
But wait, is cnxml supporting all those cases?

Reply:
This is our extension to CNXML, and there isn't even any document documenting it :) But the idea was that you can set any valid case.